kong/db/strategies/off/init.lua

```diff
@@ -333,8 +328,26 @@ local function select(self, pk, options)
   local schema = self.schema
   local ws_id = workspace_id(schema, options)
   local pk = pk_string(schema, pk)
+
+  -- if no specific ws_id is provided, we need to search all workspace ids
+  if ws_id == "*" then
```

Comment:
should define a constant for `*`

Reply:
could export from `db/declarative/import.lua`

Reply:
```
 $ git diff kong/db/declarative/import.lua
diff --git a/kong/db/declarative/import.lua b/kong/db/declarative/import.lua
index 3d7b6fb68..1b2bfc727 100644
--- a/kong/db/declarative/import.lua
+++ b/kong/db/declarative/import.lua
@@ -616,4 +616,6 @@ return {
   load_into_cache_with_events = load_into_cache_with_events,
   insert_entity_for_txn = insert_entity_for_txn,
   delete_entity_for_txn = delete_entity_for_txn,
+
+  GLOBAL_WORKSPACE_TAG = GLOBAL_WORKSPACE_TAG,
 }
```

do you think if it is ok?

Reply:
Sure.

Reply:
fixed, thx